client/api/filesystem/v2alpha1/api.proto

```diff
@@ -86,6 +89,32 @@ message RmdirResponse {
     // Intentionally empty.
 }
 
+message RmdirContentsRequest {
+    // The path whose contents will be removed in the host's filesystem.
+    // All special characters allowed by Windows in path names will be allowed
+    // except for restrictions noted below. For details, please check:
+    // https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file
+    //
+    // Restrictions:
+    // Only absolute path (indicated by a drive letter prefix: e.g. "C:\") is accepted.
+    // Depending on the context parameter of this function, the path prefix needs
+    // to match the paths specified either as kubelet-csi-plugins-path
+    // or as kubelet-pod-path parameters of csi-proxy.
+    // UNC paths of the form "\\server\share\path\file" are not allowed.
+    // All directory separators need to be backslash character: "\".
+    // Characters: .. / : | ? * in the path are not allowed.
+    // Path cannot be a file of type symlink.
+    // Maximum path length will be capped to 260 characters.
+    string path = 1;
+
+    // Force remove all contents under path (if any).
+    bool force = 2;
```

Comment:
why do we want to have this option? If force is false, it can only remove files, or empty dirs? 
Does RmdirContents mean we want to remove everything under it, not just empty dirs?

Reply:
good observation, from LVP it's always called with `force = true` and if it's false then it's like you say for, this is because I copied the message from the existing `Rmdir` message

I removed the field